pkg/agent/packetcapture/packetcapture_controller.go

```diff
@@ -176,15 +178,15 @@ func (c *Controller) enqueuePacketCapture(pc *crdv1alpha1.PacketCapture) {
 func (c *Controller) Run(stopCh <-chan struct{}) {
 	defer c.queue.ShutDown()
 
-	klog.InfoS("Starting packetcapture controller", "name", controllerName)
-	defer klog.InfoS("Shutting down packetcapture controller", "name", controllerName)
+	klog.InfoS("Starting controller", "name", controllerName)
+	defer klog.InfoS("Shutting down controller", "name", controllerName)
 
 	cacheSynced := []cache.InformerSynced{c.packetCaptureSynced}
 	if !cache.WaitForNamedCacheSync(controllerName, stopCh, cacheSynced...) {
 		return
 	}
 
-	err := defaultFS.MkdirAll(packetDirectory, 0755)
+	err := defaultFS.MkdirAll(packetDirectory, 0700)
```

Comment:
just to confirm: the file can still be copied out of the Pod with `kubectl cp`, for the case where sftp is not used?
I believe this directory only exists in the container filesystem (it's not available on the Node), so there is no visible impact of this change?